### ISSUE TITLE
[Bug] Kubecost stops collecting data

### ISSUE THREAD

**Issue by magga2001** [kubecost/cost-analyzer-helm-chart#3935]
### Kubecost Helm Chart Version

2.6.3

### Kubernetes Version

2.6.3

### Kubernetes Platform

EKS

### Description

Hi, I have kubecost opensource installed on one of my EKS clusters. However, the data just suddenly stop collecting one day. It was collecting nicely for the past 3 weeks, then it just suddenly stop collecting. Uninstalling helm chart and reinstalling didn't help too. Everything seems to be running fine.

I can access Kubecost but it doesn't detect the cluster at all. Running the allocation API just return success without any data

Also, running the same helm chart and value to other cluster has no issue and work normally.

### Steps to reproduce

1. To be honest, I am not sure, it just stop working.

### Expected behavior

Kubecost collect data and cluster is detected within kubecost

### Impact

I cannot collect data

### Logs

```Shell
DBG ETL: Allocation: QueryAllocation([2025-03-10T00:00:00+0000, 2025-03-13T00:00:00+0000), [cluster node namespace pod container]) from StoreDriver[1d] 139.231µs

DBG Prometheus vector does not have cluster id
```

### Troubleshooting

- [x] I have read and followed the [issue guidelines](https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/ISSUE_GUIDELINES.md) and this is a bug impacting only the Helm chart.
- [x] I have searched other issues in this repository and mine is not recorded.

**Comment by chipzoller** [comment]
> I have kubecost opensource installed

Is this Kubecost Free (not open source) or OpenCost (open source)?

**Comment by magga2001** [comment]
Free version

**Comment by chipzoller** [comment]
Then per the [issue guidelines](https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/ISSUE_GUIDELINES.md), please open a ticket at support.kubecost.com as this is not a Helm chart issue.
